.github/workflows/book-weekly.yml

```diff
@@ -57,4 +57,4 @@ jobs:
           restore-keys: ${{ runner.os }}-openml-
 
       - name: Render book
-        run: quarto render book/ --cache-refresh
+        run: quarto render book/ --cache-refresh --execute-debug
```

Comment:
Is this for the current run only or do we want to enable this permanently?
If it causes some noticeable overhead or very verbose output we might want to remove it once everything's settled again I guess?

Reply:
hmm, good point about the overhead, it did not seem to drastically increase rendering times and it created some very useful debug info

Reply:
@jemus42 are you happy for this to be merged then?

Reply:
Sure - if it ends up being superfluous, we can still disable debug output down the road, I just wanted to check that's not left in on accident :)